src/command.js

```diff
@@ -31,7 +32,14 @@ module.exports = class Command {
         });
         Rx.fromEvent(child, 'close').subscribe(([exitCode, signal]) => {
             this.process = undefined;
-            this.close.next(exitCode === null ? signal : exitCode);
+            this.close.next({
+                command: this.command,
+                name: this.name,
+                prefixColor: this.prefixColor,
+                env: this.env,
+                index: this.index,
+                exitCode: exitCode === null ? signal : exitCode,
```

Comment:
Sounds good, I was considering that too. 👍 I even tried keeping these command info fields in a `command` field in `Command` so any new fields would automatically be included, but that seemed to require even more changes elsewhere. 😄

How do you feel about `index`, should it be inside `command` or on its own at the top level?

Reply:
I kept `index` at the top level as it's not in the original command fields. I pushed the changes necessary to change the shape of the returned value.

I tried again to change the code internally so that it would automatically pick up any changes in the "command info", but it seems to require changes in so many places that it's maybe not worth it.